input/script_utils.go

```diff
@@ -104,11 +106,11 @@ func SpendMultiSig(witnessScript, pubA, sigA, pubB, sigB []byte) [][]byte {
 	// ensure the signatures appear on the Script Virtual Machine stack in
 	// the correct order.
 	if bytes.Compare(pubA, pubB) == 1 {
-		witness[1] = sigB
-		witness[2] = sigA
+		witness[1] = append(sigB.Serialize(), byte(txscript.SigHashAll))
+		witness[2] = append(sigA.Serialize(), byte(txscript.SigHashAll))
 	} else {
-		witness[1] = sigA
-		witness[2] = sigB
+		witness[1] = append(sigA.Serialize(), byte(txscript.SigHashAll))
```

Comment:
Solid change to move the sighash application here as well, cuts down on some clutter elsewhere when this was done manually.